spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql-r2dbc/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>spring-cloud-gcp-starters</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>2.0.7-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-cloud-gcp-starter-sql-r2dbc</artifactId>
+    <name>Spring Cloud GCP Starter - R2DBC</name>
```

Comment:
Why only one starter for both MySQL and Postgres R2DBC? We made the decision to split them for JDBC. The problem with combining them is that you're automatically bringing in dependencies that will not be needed.

Reply:
You make a valid point here. Introduced two starters (one for MySQL and PostgreSQL).